javascript/javascript3/week1/giphy.js

```diff
@@ -0,0 +1,77 @@
+let sumbitBtn = document.getElementById("sub");
+let empty = document.getElementById("empty");
+
+
+
+let apiUrl =
+  "http://api.giphy.com/v1/gifs/search?&api_key=UzMWGrXRdD8bLvzF5UnZerMOmKRAVVZI&q="; ;
+let gifNumber = document.getElementById("num");
+let searchWord = document.getElementById("search");
+let imgWrapper = document.querySelector("#imgWrapper")
+
+
+const search = sumbitBtn.onclick=(e)=>{
+ e.preventDefault();
+
+ if (searchWord.value == null || searchWord.value.trim() == "") {
```

Comment:
Always do check with === instant of == 

https://codeburst.io/javascript-double-equals-vs-triple-equals-61d4ce5a121a

Reply:
you can also do a direct check if (searchWord.value) { }

Reply:
i tried to write if(searchWord.value){} , but it didn't work